docker-demo/set_welcome_message.groovy

```diff
@@ -0,0 +1,8 @@
+jenkins.model.Jenkins.instance.setSystemMessage(
+"""
+<p>Welcome to Jenkins Blue Ocean Demo. Click on the image below: </p>
+<p><a href="/blue"><img src="/userContent/blueocean-demo.png" /></a></p>
+
+<p>log in with <em>admin/admin</em></p>
```

Comment:
@ndeloof as per the comment posted in #215, could you maybe add \<strong> around that line, so that users have bigger chances to read it and know the admin user credentials? Thanks